Explorer/Assets/DCL/Communities/CommunitiesCard/CommunityCardView.cs

```diff
@@ -101,6 +118,51 @@ private void Awake()
             membersTextButton.onClick.AddListener(() => ToggleSection(Sections.MEMBERS));
             placesButton.onClick.AddListener(() => ToggleSection(Sections.PLACES));
             placesWithSignButton.onClick.AddListener(() => ToggleSection(Sections.PLACES));
+            placesShortcutButton.onClick.AddListener(() => OpenWizardRequested?.Invoke());
+
+            contextMenu = new GenericContextMenu(contextMenuSettings.ContextMenuWidth,
+                              offsetFromTarget: contextMenuSettings.OffsetFromTarget,
+                              verticalLayoutPadding: contextMenuSettings.VerticalPadding,
+                              elementsSpacing: contextMenuSettings.ElementsSpacing,
+                              anchorPoint: ContextMenuOpenDirection.BOTTOM_LEFT)
+                         .AddControl(new ButtonContextMenuControlSettings(contextMenuSettings.LeaveCommunityText, contextMenuSettings.LeaveCommunitySprite, () => LeaveCommunityRequested?.Invoke()))
+                         .AddControl(deleteCommunityContextMenuElement = new GenericContextMenuElement(new ButtonContextMenuControlSettings(contextMenuSettings.DeleteCommunityText, contextMenuSettings.DeleteCommunitySprite, OnDeleteCommunityRequested)));
+        }
+
+        private void OnDeleteCommunityRequested()
+        {
+            confirmationDialogCts = confirmationDialogCts.SafeRestart();
+            ShowDeleteConfirmationDialogAsync(confirmationDialogCts.Token).Forget();
```

Comment:
Same feedback as above: don't Forget or swear that it will handle everything forever.